### PR TITLE
Kirilushkin Aleksandr Tasks 71-80

### PR DESCRIPTION
**Pull request by KirilushkinAleksandr**

### CODE REVIEW

File: tasks/71_binary-encode/binary-encode.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function binaryEncode(input) {
+  if (!input) {
+    return '';
+  }
+  const chars = input.split('');
+  const res = chars.map((item, i) => {
+    const binary = parseInt(item.charCodeAt(0)).toString(2);
+    return '0'.repeat(8 - binary.length) + binary;
+  }
```
Comment (alexkond) [inline tasks/71_binary-encode/binary-encode.js]: `});` should be on the same line

**Review by alexkond — CHANGES_REQUESTED** [review]

File: tasks/72_carrying/carrying.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * // return 5
+ * addTogether(2, 3)
+ * function addTogether sums two arguments together
+ * @param  {...any} input array of passed numbers
+ * @return {undefined} if any of passed data is not a number
+ * @return {Number} sum of numbers
+ * @return {Function} if single value is passed
+ */
+function addTogether(...input) {
+  if (typeof(input[0]) !== 'number' ) {
```
Comment (vovcyan) [inline tasks/72_carrying/carrying.js]: (typeof(input[0]) !== 'number')

File: tasks/72_carrying/carrying.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    return;
+  }
+  let sum = input[0];
+  if (!input[1]) {
+    return (input) => {
+      if (typeof(input) !== 'number' ) {
```
Comment (vovcyan) [inline tasks/72_carrying/carrying.js]: Same

File: tasks/72_carrying/carrying.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        return;
+      }
+      return sum + input;
+    };
+  }
+  if (typeof(input[1]) !== 'number' ) {
```
Comment (vovcyan) [inline tasks/72_carrying/carrying.js]: Same

**Review by alexkond — DISMISSED** [review]

File: tasks/76_deep-object-clone/deep-object-clone.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+ * (nested values are affected too) passed object
+ * @param {Object} inputObj, object passed for cloning
+ * @return {Object} deep copy of an original object
+ */
+function deepClone(inputObj) {
+  const clone = {};
+  for (let key in inputObj) {
+    if (typeof inputObj[key] === 'object') {
+      clone[key] = deepClone(inputObj[key]);
+    } else clone[key] = inputObj[key];
```
Comment (vovcyan) [inline tasks/76_deep-object-clone/deep-object-clone.js]: {
 clone[key] = inputObj[key];
}

File: tasks/79_calculate_primoral_of_a_number/calculate_primoral_of_a_number.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+      if (i % j === 0) {
+        isPrime = false;
+      }
+    }
+    if (isPrime) {
+      res *= i;
+      count--;
+    }
+    isPrime = true;
+    if (count === 0) {
```
Comment (vovcyan) [inline tasks/79_calculate_primoral_of_a_number/calculate_primoral_of_a_number.js]: !count

File: tasks/80_sum-of-prime-indexes/sum-of-prime-indexes.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+function sumPrimeIndexedElements(inputArr) {
+  const res = inputArr.slice();
+  return res.reduce(
+      (primesSum, item, index) => {
+        if (isPrime(index)) {
+          return primesSum + item;
+        }
+        return primesSum;
+      },
+      0
```
Comment (vovcyan) [inline tasks/80_sum-of-prime-indexes/sum-of-prime-indexes.js]: Weird formatting

**Review by vovcyan — CHANGES_REQUESTED** [review]

**Review by vovcyan — APPROVED** [review]

**Review by alexkond — APPROVED** [review]

**Pull request merged**